tests/ballerina-spec-conformance-tests/src/test/java/org/ballerinalang/test/TestRunner.java

```diff
@@ -112,4 +113,9 @@ public void generateReports(ITestContext context) throws IOException {
             return testCases.iterator();
         }
     }
+
+    private Set<String> getPredefinedLabels() {
```

Comment:
Every time we call this we dynamically populate the predefined labels. 
Instead, shall we have a static final variable to keep the predefined labels, so we populate it only once.
```java
private static final Set<String> predefinedLabels = TestRunnerUtils.readLabels(TEST_DIR.toString()).keySet();
```